### ISSUE TITLE
Question about the use of auxiliary_frame in Depth and Semantic task

### ISSUE THREAD

**Issue by yanchi-3dv** [LinShan-Bin/OccNeRF#22]
Thank you for your excellent work, I would like to confirm if you do not use auxiliary_frame and depth auxiliary_frame in sementic training for your work.

**Comment by LinShan-Bin** [comment]
Yes, we do not use auxiliary_frame and depth auxiliary_frame in sementic training.
